src/disk-images/github-runner-linux/root/etc/sysconfig/network/ifcfg-eth0

```diff
@@ -0,0 +1,4 @@
+BOOTPROTO='dhcp'
+MTU=''
```

Comment:
Are `MTU` and `REMOTE_IPADDR` necessary?

Reply:
From trial and error: they're not, removed. (This was just copied from a running machine, hence the empty settings.)